scripts/get

```diff
@@ -213,7 +213,9 @@ while [[ $# -gt 0 ]]; do
     '--version'|-v)
        shift
        if [[ $# -ne 0 ]]; then
-           export DESIRED_VERSION="${1}"
+          # FIXME(bacongobbler): hard code the desired version for the time being.
+          # A better fix would be to filter for Helm 2 release pages.
+           export DESIRED_VERSION="${1:="v2.16.1"}"
```

Comment:
This looks suspicious.
The `${1:="v2.16.1"}` form will set the `v2.16.1` string only if `$1` is unset, however, we are in an if statement that checked that `$1` was not null.
Maybe forcing the version should instead be done in `checkDesiredVersion()`?

When testing this code change, things might still have worked since 2.16.1 is actually the latest version :-)

Reply:
Thank you for catching this. I'll see what I can do.